### PR TITLE
Fix password validation and checkbox style

### PR DESCRIPTION
**Pull request by arianeleonard**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix

## Description

Update checkbox style to handle color change.
Show password validation hints when no password is entered.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [x] Tested on all relevant platforms

## Internal Issue (If applicable):

https://dev.azure.com/nventive/Practice%20committees/_backlogs/backlog/Mobile%20(.Net)/Features/?workitem=255989
https://dev.azure.com/nventive/Practice%20committees/_backlogs/backlog/Mobile%20(.Net)/Features/?workitem=255990

### CODE REVIEW

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/app/ApplicationTemplate.Shared/Presentation/Authentication/PasswordFormViewModel.cs [hunk 1]
```diff
@@ -19,16 +19,19 @@ public string Password
 		public bool? PasswordHasMinimumLength
 		{
 			get => this.GetFromObservable(ObservePasswordHasMinimumLength(), initialValue: null);
+			set => this.Set(value);
```
[the remark this answers is not in the document]
Comment (arianeleonard) [reply src/app/ApplicationTemplate.Shared/Presentation/Authentication/PasswordFormViewModel.cs:22]: The observables return null when the password is null or empty. Though, for this specific case, we need them to return false when the password is null or empty.

Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Shared/Presentation/Authentication/PasswordFormViewModel.cs:22]: You should use private setters here :)

File: src/app/ApplicationTemplate.Shared/Presentation/Authentication/CreateAccountPageViewModel.cs [hunk 2]
```diff
@@ -37,6 +37,10 @@ public class CreateAccountPageViewModel : ViewModel
 
 				await this.GetService<ISectionsNavigator>().SetActiveSection(ct, "Home", () => new DadJokesPageViewModel());
 			}
+			else
```
[the remark this answers is not in the document]
Comment (arianeleonard) [reply src/app/ApplicationTemplate.Shared/Presentation/Authentication/CreateAccountPageViewModel.cs:40]: There is validation in the CreateAccountFormViewModel.

[the remark this answers is not in the document]
Comment (jeanplevesque) [reply src/app/ApplicationTemplate.Shared/Presentation/Authentication/CreateAccountPageViewModel.cs:40]: Right! I just realized that there were 2 separate form VMs.

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**